pkg/cloud/services/eks/nodegroup.go

```diff
@@ -460,8 +482,22 @@ func (s *NodegroupService) reconcileNodegroupConfig(ng *eks.Nodegroup) error {
 		return errors.Wrap(err, "created invalid UpdateNodegroupConfigInput")
 	}
 
+	conditions.MarkFalse(
+		s.scope.ManagedMachinePool,
+		expinfrav1.EKSNodegroupUpdateSucceededCondition,
+		expinfrav1.EKSNodegroupUpdatingReason,
+		clusterv1.ConditionSeverityInfo,
+		"",
+	)
 	_, err = s.EKSClient.UpdateNodegroupConfig(input)
 	if err != nil {
+		conditions.MarkFalse(
+			s.scope.ManagedMachinePool,
+			expinfrav1.EKSNodegroupUpdateSucceededCondition,
+			expinfrav1.EKSNodegroupFailedToUpdateReason,
+			clusterv1.ConditionSeverityInfo,
+			"",
```

Comment:
Probably best to set the error message here

Reply:
Updated